Add Flair indexing (#1612)
ava-labs/avalanche-docs: docs/tooling/indexers.md

- [Avalanche Quickstart](https://academy.subquery.network/quickstart/quickstart_chains/avalanche.html)
- [Mainnet Starter Project](https://github.com/subquery/ethereum-subql-starter/tree/main/Avalanche/avalanche-starter)
- [Fuji Starter Project](https://github.com/subquery/ethereum-subql-starter/tree/main/Avalanche/avalanche-fuji-starter)

### Flair

> [Flair](https://flair.dev) is a real-time and historical custom data indexing for any evm chain.

Reviewer: ```suggestion
[Flair](https://flair.dev) provides real-time and historical custom data indexing for any EVM-compatible chain.
```
